controls/radgridview/features/overview-controlpanel.md

```diff
@@ -65,4 +65,98 @@ For example:
 
 
 >tipYou can download a __runnable solution__ showing how to define ControlPanelItems from our online SDK repository [here](https://github.com/telerik/xaml-sdk/), the example is listed under __GridView/ControlPanel__.
-    
+   
+
+## Reusing Control Panel Items  ##
+
+As of __Q1 2016__ we have added a DataTemplate support for the __ButtonContent__ and __Content__ properties of the __ControlPanelItem__. Developers have the ability to add a common Style targeting the ControlPanel that all the RadGridView controls in the application will inherit.
```

Comment:
As of Q1 2016 RadGrdiView supports DataTemplates for the ButtonContent and Content properties of the ControlPanelItem. You have the ability...